rector_examples/datetime_datetime_storage_format.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * This demonstrates the deprecated static calls that might be called from procedural code like `.module` files.
+ */
+
+/**
+ * A simple example.
+ */
+function simple_example() {
+  $format = DATETIME_DATETIME_STORAGE_FORMAT;
+
+}
+
+/**
+ * An example using the constant as an argument.
+ */
+function as_an_argument() {
+  $date = new DrupalDateTime('now', new \DateTimezone('America/Los_Angeles'));
```

Comment:
Can you use a `use` statement or FQCN here for `DrupalDateTime`?

Reply:
Sorry, I misunderstood. Thought you were talking about the new interfacve. Didn't notice DrupalDateTime. Fixed in 734a30c.